app/services/webhook/dispatch_event.rb

```diff
@@ -0,0 +1,28 @@
+module Webhook
+  class DispatchEvent
+    def initialize(event_type, record)
+      @event_type = event_type
+      @record = record
+    end
+
+    def self.call(*args)
+      new(*args).call
+    end
+
+    def call
+      endpoint_urls = Endpoint.for_events([event_type]).pluck(:target_url)
+      return if endpoint_urls.empty?
+
+      # decorating articles, may need to change the logic when sending other types of objects
```

Comment:
Currently, I'm figuring out where the `decorate` should be, and fixing the build.